front/src/weekview/Weekview.js

```diff
@@ -320,7 +320,8 @@ class Weekview extends Component {
       info = false;
       if (this.state.paivat[j].tracks) {
         this.state.paivat[j].tracks.forEach((track) => {
-          if (track.notice !== null) {
+          if (track.notice !== null && track.notice !== "") {
+            Attention = track.notice;
```

Comment:
Line 324:13:  'Attention' is not defined  no-undef

Project doesn't run.

Reply:
Interesting, I think there was a merge conflict with an old version of develop which dazzled me with what wasn't supposed to actually be there. Need to be careful in the future with this. Fixed.